seatunnel-connectors-v2/connector-jdbc/pom.xml

```diff
@@ -30,7 +30,7 @@
     <name>SeaTunnel : Connectors V2 : Jdbc</name>
 
     <properties>
-        <mysql.version>8.0.16</mysql.version>
+        <mysql.version>8.0.27</mysql.version>
```

Comment:
What is the reason for changing this version

Reply:
https://debezium.io/releases/1.6/

debezium using 8.0.27

Reply:
https://issues.redhat.com/plugins/servlet/mobile#issue/DBZ-4286